### DISCUSSION TITLE
Read ME error

### DISCUSSION THREAD

**Discussion by deniszakarovni** [community/community#102822] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

why my readme sometimes error ? it doesn't add the stats and also not show the animation , any solution?

**Comment by naufalanasea** [comment, marked as answer]
just refresh it , it is normal sometimes the readme does not appear properly

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @deniszakarovni! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
